circle.yml

```diff
@@ -267,13 +313,61 @@ jobs:
       - deploy:
           name: Publish Navigation SDK 1.0 to Bintray
           command: |
-            if [[ $CIRCLE_BRANCH == master ]]; then
-              version=$(cat gradle/artifact-settings.gradle | grep "versionName")
-              if [[ $version != *"SNAPSHOT"* ]]; then
-                make 1.0-publish-to-bintray
-              else
-                make 1.0-publish-to-artifactory
-              fi
+            version=$(cat gradle/artifact-settings.gradle | grep "versionName")
+            if [[ $version != *"SNAPSHOT"* ]]; then
+              make 1.0-publish-to-bintray
+            fi
+
+  release-1.0-qa:
+    working_directory: ~/code
+    docker:
+      - image: mbgl/61abee1674:android-ndk-r18
+    environment:
+      BINTRAY_REPO: mapbox_private
+    steps:
+      - checkout
+      - run:
+          name: Build Navigation SDK 1.0 (QA release)
+          command: make 1.0-build-release
+      - run:
+          name: Generate documentation
+          command: make javadoc-dokka
+      #     Uncomment when API' docs will be ready
+      #     - run:
+      #         name: Check public documentation
+      #         command: sh scripts/dokka-validate.sh
+      - deploy:
+          name: Publish Navigation SDK 1.0 to Bintray
+          command: |
+            version=$(cat gradle/artifact-settings.gradle | grep "versionName")
+            if [[ $version != *"SNAPSHOT"* ]]; then
+              make 1.0-publish-to-bintray
+            fi
```

Comment:
This whole flow can be extracted to a single job that accepts the Bintray repo argument.